### ISSUE TITLE
Create SPPKG?

### ISSUE THREAD

**Issue by bxlogic** [yapici/SharePoint-Based-Inventory-System#2]
Hello Engin,

can you please create SPPKG file, in that way people can install it directly without having a need to compile it, because not everyone knows the processes of compiling it :)

Thank you!

**Comment by tayram** [comment]
I would agree. I have tried 3 times in the past and never had any success.

**Comment by yapici** [comment]
Thank you for creating this issue @bxlogic. It's been a while since I worked on this project and I don't know how to create a SPPKG file. If you do and submit a pull request, I will be happy to review and merge it.

**Comment by tayram** [comment]
@yapici , give this a look: https://github.com/gunjandatta/spfx-upgrade

There is a well documented Wiki in the repo. Basically, this is a tutorial of how to take Classic SharePoint solutions, and bundle them into the Modern SPFX framework.

Cheers!

**Comment by bxlogic** [comment]
Hello Engin,

Thank you for your response. The application idea you've presented is truly remarkable, as it is both exceptionally useful and user-friendly. Introducing an SPPGK file feature would undoubtedly enhance its appeal for all users. I understand that you haven't been working on this project for very long, but with the extensive capabilities that Microsoft 365 / SharePoint now provides, there are numerous possibilities. I would be delighted to assist you, even though I may not be well-versed in the technical aspects. I can offer my support in terms of graphic design or CSS to enhance its aesthetics and keep it current with the latest trends.
